Orange/widgets/data/owfeatureconstructor.py

```diff
@@ -330,6 +339,111 @@ def data(self, index, role=Qt.DisplayRole):
             return super().data(index, role)
 
 
+def freevars(exp, env):
```

Comment:
This function has been moved above the class definition: it is called from `migrate_context`, which is called from meta class, which is sets up the class. Function below the class definition are not yet defined at that moment.